doc/news/changes/incompatibilities/20200603Munch

```diff
@@ -0,0 +1,5 @@
+Changed: The binary representation of the Triangulation and DoFHandler classes 
+created with release 9.2 cannot be read anymore due to major internal changes
```

Comment:
Can you clarify that you are, in particular, talking about `Triangulation::save()`, `Triangulation::load()`, `GridIn::read_vtk` and `GridOUt::write_vtk` to give an example?

Reply:
To be precise, it is `GridIn::read_vtu` and `GridOUt::write_vtu`

Reply:
@masterleinad @agrayver I have addressed the comments!